torrent/logging.go

```diff
@@ -6,31 +6,37 @@
  *
  */
 
-package main
+package torrent
 
 import (
 	"log"
 )
 
-func loginfo(fmt string, args ...interface{}) {
+var VERBOSITY uint32 = 0
```

Comment:
So you created the Verbosity type before, but don't use it here? Also, I don't think anything ever calls `SetLoggingVerbosity` below?